uktrade/data-hub-frontend
# Add layout for OMIS edit forms and other cleanup

## Description of change

This is a cleanup PR before the work starts to reactify the OMIS work order page and delete the legacy OMIS edit app.

The following changes have been made:
- To reduce duplication, a new layout component has been created for the OMIS edit forms. This is because the structure of the breadcrumbs is identical for all of these pages.
- The `EditAssignees` and `EditSubscribers` forms have been refactored to use React Router. These pages have also been moved to the `modules` directory.
- All legacy/redundant code associated with the above two pages has been removed.

## Test instructions

Go to any OMIS edit form. The heading/breadcrumbs should render as normal.

Go to an OMIS order and edit the Advisers in the market or Advisers in the UK. This should work as previously.

## Screenshots

No visual changes.

## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
